src/scss/components/_footer.scss

```diff
@@ -6,157 +6,175 @@
 **/
 
 .footer {
-  @include media(medium) {
-    max-height: map-deep-get($footerSize, mini, size) + px;
-  }
-  display: flex;
-  flex-direction: column;
-  height: 100%;
-  text-align: center;
-  width: 100%;
-
-  &--sticky {
-    bottom: 0;
-    justify-content: flex-end;
-    left: 0;
-    max-height: map-deep-get($footerSize, large, size) + px;
-    position: absolute;
-  }
+  background-color: get-color(wall);
+  height: auto;
 
-  &__company-info {
-    background-color: get-color(blackNeutral);
-    padding: {
-      bottom: 24px;
-      top: 24px;
-    };
+  &__title {
+    color: get-color(rastaGray);
+    display: block;
+    font-size: 12px;
+    font-weight: 900;
+    text-align: left;
+    text-transform: uppercase;
+    width: 100%;
   }
 
-  &__company-disclaimer {
-    @include media(medium) {
-      text-align: inherit;
+  &__block {
+    @include media(small) {
+      align-content: flex-start;
+      margin-top: get-space(box, large);
     }
+    align-content: flex-start;
```

Comment:
Essas propriedades não teria que estar antes do `breakpoint`?

Reply:
@williamcosta our sass lint required mixins before any declaration. Let's discuss this on our #frontend channel :)